test/common-algorithms/sorting/parallel/parallel_quicksort.chpl

```diff
@@ -0,0 +1,62 @@
+//var arr: [1..10] int = (8, 14, -8, -9, 5, -9, -3, 0, 17, 19);
+
+proc partition(arr, left: int, right: int) : int{
+	var x : int = arr(right);
+	var i : int = (left - 1);
+ 
+	for j in (left..right-1){
+		if (arr(j) <= x)
+        	{
+            		i+=1;
+            		var temp : int = arr(j);
+			arr(j) = arr(i);
+			arr(i) = temp;
```

Comment:
We actually have a swap operator, `<=>` (listed under [Assignments](http://chapel.cray.com/spec/quickReference.pdf)), that you can use instead of the explicit swap.